HTML/QuickForm/Rule/Range.php

```diff
@@ -45,7 +45,7 @@ class HTML_QuickForm_Rule_Range extends HTML_QuickForm_Rule
      * @access    public
      * @return    boolean   true if value is valid
      */
-    function validate($value, $options)
+    function validate($value, $options = null)
```

Comment:
change 2

Reply:
So this is weird - it doesn't match the parent signature - but it didn't before either - feels smelly but I guess....


```
   /**
    * Validates a value
    *
    * @access public
    * @abstract
    */
    function validate($value)
    {
        return true;
    }

```